src/disnake/ext/plugins/plugin.py

```diff
@@ -172,11 +171,14 @@ class Plugin(t.Generic[BotT]):
     logger: Optional[Union[:class:`logging.Logger`, :class:`str`]]
         The logger or its name to use when logging plugin events.
         If not specified, defaults to `disnake.ext.plugins.plugin`.
+    extras: Dict[:class:`str`, Any]
```

Comment:
This doesn't quite check out-- `Parameters` are the `__init__` parameters, and `extras` is not one of them as you used `**kwargs` there.

I think it'd be most clear if we used `**extras` both in the docstring here and in the method signature?

Reply:
Good catch, thanks! Changed in 33ac4987086a8bc07e8877b0455c557c0aedbb0b